test/groovy/ParallelDockerUpdatecliStepTests.groovy

```diff
@@ -163,10 +180,78 @@ class ParallelDockerUpdatecliStepTests extends BaseTest {
     // And updatecli(action: 'diff') is called
     assertTrue(assertMethodCallContainsPattern('updatecli', 'action=diff'))
 
+    // No Warning issued
+    assertFalse(assertMethodCallContainsPattern('echo', 'WARNING:'))
+
     // And the custom parameters are taken in account
     assertTrue(assertMethodCallContainsPattern('buildDockerAndPublishImage', 'mainBranch=' + anotherMainBranchName))
     assertTrue(assertMethodCallContainsPattern('string', anotherCredentialsId))
     assertTrue(assertMethodCallContainsPattern('updatecli', 'cronTriggerExpression=' + anotherCronTriggerExpression))
     assertTrue(assertMethodCallContainsPattern('updatecli', 'containerMemory=' + anotherContainerMemory))
   }
+
+  @Test
+  void itRunsSuccessfullyWithCustomUpdatecliConfig() throws Exception {
+    def script = loadScript(scriptName)
+
+    addEnvVar('BRANCH_IS_PRIMARY', 'true')
+    script.call(
+      imageName: testImageName,
+      updatecliConfig: [
+        containerMemory: anotherContainerMemory,
+      ],
+    )
+    printCallStack()
+
+    // Then we expect a successfull build
```

Comment:
No "When" above?

Reply:
Same for L238

Reply:
should be better with last version, WDYT?